HomeWork 5/LoggerAplication/LoggerAplication/Logger.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Text;
+
+namespace LoggerAplication
+{
+    internal class Logger
+    {
+        private DateTime _logTime;
+
+        private string _messageText = string.Empty;
+
+        private LogType _logType;
+
+        private StringBuilder _logs = new StringBuilder();
+
+        private static Logger _loggerInstance = null;
+
+        private Logger() { }
+
+        public static Logger GetInstance()
+        {
+            if (_loggerInstance == null)
+            {
+                _loggerInstance = new Logger();
+            }
+
+            return _loggerInstance;
+        }
+
+        public void Log(string message, LogType logType)
+        {
+            _logTime = DateTime.Now;
+
+            _logType = logType;
+
+            _messageText = message;
+
+            DisplayLogToConsole();
+
+            WritteToLoglist();
+        }
+
+        private void DisplayLogToConsole() => Console.WriteLine($"{_logTime} : {_logType} : {_messageText}");
+
+        public string ShowLoglist() => _logs.ToString();
+
+        internal void SaveLog() => File.WriteAllText("log.txt", ShowLoglist());
+
+        private void WritteToLoglist() => _logs.Append($"{_logTime} : {_logType} : {_messageText}\n");
```

Comment:
I see we have the same format on line 42 and 48; therefore, we need to think on how to use one variable for both cases

Reply:
)) didn`t seen that